lms/static/sass/discussion/_layouts.scss

```diff
@@ -31,4 +31,8 @@
 
 .discussion-column {
     min-height: 500px;
+
+    .new-post-article {
```

Comment:
I discovered yesterday that the entire form is duplicated in the edit post modal too (thread-edit.underscore). I think you'll need to make the same changes there too.

If you felt like reducing tech debt, you could also remove thread-edit.underscore and have a single template for both adding and editing posts. No worries if you don't, because I didn't. :-)